src/main/java/techreborn/tiles/generator/TileSolarPanel.java

```diff
@@ -113,16 +110,23 @@ public boolean canProvideEnergy(final EnumFacing direction) {
 
 	@Override
 	public double getBaseMaxOutput() {
-		return maxOutput;
+		return 0;
+		//TODO Figure out why panel.generationrateN crashes the game on load.
 	}
 
 	@Override
 	public double getBaseMaxInput() {
 		return 0;
 	}
 
+		@Override
+		public EnumPowerTier getBaseTier() {
+			return EnumPowerTier.MEDIUM;
+			//TODO Figure out why panel.powerTier crashes the game on load.
```

Comment:
@drcrazy I need help with this, I am not sure why it is causing crashes. I removed the internal buffer as requested by modmuss and pulled directly from the ENUM but only some of it is working. Any ideas?